lua/litee/symboltree/init.lua

```diff
@@ -367,7 +367,7 @@ function M.setup(user_config)
     -- merge in config
     if user_config ~= nil then
         for key, val in pairs(user_config) do
-            config[key] = val
+          config[key] = "table" == type(config[key]) and nil ~= val and vim.tbl_extend("keep", val, config[key]) or val
```

Comment:
Think we can break this into multiple statements? It's a lot going on for a single line.